release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -521,6 +521,19 @@ registry.redhat.io/rhpam-7/rhpam-businesscentral-rhel8:7.9.0
 registry.redhat.io/rhpam-7/rhpam-smartrouter-rhel8:7.9.0
 ----
 
+[id="ocp-4.8-sandboxed-containers"]
+=== {sandboxed-containers-first}
+
+[id="ocp-4.8-sandboxed-containers-tp"]
+==== {sandboxed-containers-first} support on {product-title} (Technology Preview)
+
+{sandboxed-containers-first} 1.0.0 Technology Preview release introduces built-in support for running Kata Containers as an additional runtime. {sandboxed-containers-first} enables users to choose Kata Containers as an additional runtime to provide additional isolation for their workloads. The {sandboxed-containers-operator} enables users to perform tasks such as installation, removal, updates, and monitoring installation state when describing the `KataConfig` custom resource.
+
+{sandboxed-containers-first} are only supported on bare metal. {op-system-first} is the only supported operating system for {sandboxed-containers-first} 1.0.0. Disconnected environments are not supported in {product-title} {product-version}.
```

Comment:
The version appears as `Branch Build` sometimes.

Reply:
@zanetworker `Branch Build` will result in the 4.8 product version showing up.